.eslintrc

```diff
@@ -36,8 +40,6 @@
     }
   },
   "rules": {
-    "arrow-parens": [2, "as-needed"],
-    "indent": [2, 2, { "SwitchCase": 1, "VariableDeclarator": 2, "MemberExpression": 2, "ObjectExpression": 1, "CallExpression": { "arguments": 2 } }],
```

Comment:
These rules are controlled by Prettier. Since we use the same Prettier config as amphtml, I imagine we're probably satisfied with the results. However, maybe we want to copy the `.eslintrc` rules from amphtml over as well? I'll run `--fix` once with and once without the amphtml ruleset, and will take a look at the diff.

Reply:
I can think of several `amphtml` lint rules that we likely don't care about here. For example, the JSDoc rules that we primarily need for closure compiler type checking and optimization. Bringing in the formatting rules SGTM, though.